src/pkgcheck/checks/metadata.py

```diff
@@ -252,6 +252,19 @@ class BannedEapi(_EapiResult, results.Error):
     _type = 'banned'
 
 
+class StableKeywordsOnTestingEapi(results.VersionResult, results.Error):
+    """Package has stable keywords on EAPI marked as testing-only."""
+
+    def __init__(self, eapi, keywords, **kwargs):
+        super().__init__(**kwargs)
+        self.eapi = str(eapi)
+        self.keywords = sorted(keywords)
```

Comment:
Keyword class attributes should always be immutable. If you look at other examples, the sorted() call is usually done before the instance is created, and then tuple() is used for the keyword attribute.